src/renderers/common/CubeRenderTarget.js

```diff
@@ -2,13 +2,19 @@ import { equirectUV } from '../../nodes/utils/EquirectUVNode.js';
 import { texture as TSL_Texture } from '../../nodes/accessors/TextureNode.js';
 import { positionWorldDirection } from '../../nodes/accessors/Position.js';
 import NodeMaterial from '../../materials/nodes/NodeMaterial.js';
+import { blur, getBlurParams } from '../../nodes/pmrem/PMREMUtils.js';
+import { uniform } from '../../nodes/core/UniformNode.js';
+import { uniformArray } from '../../nodes/accessors/UniformArrayNode.js';
+import { float, vec3, Fn } from '../../nodes/tsl/TSLBase.js';
 
 import { WebGLCubeRenderTarget } from '../../renderers/WebGLCubeRenderTarget.js';
 import { Scene } from '../../scenes/Scene.js';
 import { CubeCamera } from '../../cameras/CubeCamera.js';
 import { BoxGeometry } from '../../geometries/BoxGeometry.js';
 import { Mesh } from '../../objects/Mesh.js';
 import { BackSide, NoBlending, LinearFilter, LinearMipmapLinearFilter } from '../../constants.js';
+import { cubeTexture as TSL_CubeTexture } from '../../nodes/accessors/CubeTextureNode.js';
```

Comment:
This is weird - when I tried to have a local variable named `cubeTexture` it didn't work - said it was a function instead (this one). Looks like it's getting properly renamed here, but apparently not? Is it somehow globally included somewhere?